apis/fluent-en_GB/atrium-api-fluent-en_GB/src/jvmTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/DateSubjectChangerSamples.kt

```diff
@@ -48,4 +49,42 @@ class DateSubjectChangerSamples {
             }
         }
     }
+
+    @Test
+    fun asLocalDateTimeFeature() {
+        val date = formatter.parse("1995-07-17")
+
+        expect(date)
+            .asLocalDateTime() // subject is now of type LocalDate
+            .toEqual(LocalDateTime.parse("1995-07-17T00:00:00"))
+
+        fails {
+            expect(date)
+                .asLocalDate()                                  // subject is now of type LocalDate
+                .toBeAfter(LocalDate.parse("2025-07-17"))       // fails
+                .toBeBefore(LocalDate.parse("1996-07-17"))      // not evaluated/reported because `toBeAfter` already fails
+            // use `.asLocalDate { ... }` if you want all assertions evaluated
```

Comment:
```suggestion
            //                                                     use `.asLocalDateTime { ... }` if you want all assertions evaluated
```
so that `use` is below `not`, would you mind to fix that for the other samples in this in asLocalDateFeature and asLocalDate as well?